api/access.go

```diff
@@ -22,8 +27,8 @@ func Unlock(master string) error {
 	}
 	respMap := make(map[string]string)
 	if err := uri.Post(&respMap, string(contentJSON)); err != nil {
-		if init_error, present := respMap["error"]; present {
-			return errors.New(init_error)
+		if initError, present := respMap["error"]; present {
+			return errors.New(initError)
```

Comment:
Can you remove these superfluous changes?

Reply:
Hey. This is golang we're talking about. What's today simply caught by `golint` is tomorrow mandated by the compiler.

Fine...

Reply:
Actually, I realized that entire if is unnecessary because I'm handling the error parsing in Request now